.kokoro/docker/docs/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ubuntu:20.04
+from ubuntu:20.10
```

Comment:
Renovatebot doesn't quite work correctly with Ubuntu versions at the moment. We should stay on the stable release (`.04`)

https://github.com/renovatebot/renovate/issues/6060